backend/models/eventSchema.ts

```diff
@@ -0,0 +1,16 @@
+import * as mongoose from 'mongoose';
+
+export const EventSchema = new mongoose.Schema(
+  {
+    title: String,
+    start: Date,
+    end: Date,
+    location: String,
+    notes: String,
+    shifts: Array,
```

Comment:
For our purposes, we want to be more specific and have here `[ObjectId]` , rather than just Array.  
Instead, this may look something like `[mongoose.SchemaTypes.ObjectId]`  
some [docs](https://mongoosejs.com/docs/schematypes.html) for reference.